pkg/asset/agent/installconfig.go

```diff
@@ -313,48 +333,16 @@ func warnUnusedConfig(installConfig *types.InstallConfig) {
 			fieldPath := field.NewPath("Platform", "Baremetal", "ExternalBridge")
 			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ExternalBridge))
 		}
-		if baremetal.ProvisioningNetwork != defaultBM.ProvisioningNetwork {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetwork")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetwork))
-		}
-		if baremetal.ProvisioningBridge != defaultBM.ProvisioningBridge {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningBridge")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningBridge))
-		}
-		if baremetal.ProvisioningNetworkInterface != defaultBM.ProvisioningNetworkInterface {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetworkInterface")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetworkInterface))
-		}
-		if baremetal.ProvisioningNetworkCIDR.String() != defaultBM.ProvisioningNetworkCIDR.String() {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningNetworkCIDR")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningNetworkCIDR))
-		}
 		if baremetal.DeprecatedProvisioningDHCPExternal != defaultBM.DeprecatedProvisioningDHCPExternal {
 			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningDHCPExternal")
 			logrus.Warnf(fmt.Sprintf("%s: true is ignored", fieldPath))
 		}
-		if baremetal.ProvisioningDHCPRange != defaultBM.ProvisioningDHCPRange {
-			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningDHCPRange")
-			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningDHCPRange))
+		if baremetal.ProvisioningBridge != defaultBM.ProvisioningBridge {
+			fieldPath := field.NewPath("Platform", "Baremetal", "ProvisioningBridge")
+			logrus.Warnf(fmt.Sprintf("%s: %s is ignored", fieldPath, baremetal.ProvisioningBridge))
```

Comment:
I'm not sure that we need to warn the user about this (now or ever). There is no provisioning VM, so the ProvisioningBridge setting is irrelevant. But maybe this would be better handled separately as a bug and backported. I see some others in the same category (LibvirtURI, BootstrapProvisioningIP, BootstrapOSImage, BootstrapExternalStaticIP, BootstrapExternalStaticGateway, ExternalBridge).